pom.xml

```diff
@@ -198,6 +200,78 @@
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.apache.rat</groupId>
+        <artifactId>apache-rat-plugin</artifactId>
+        <version>0.12</version>
+        <executions>
+          <execution>
+            <id>rat-checks</id>
+            <phase>validate</phase>
+            <goals>
+              <goal>check</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <skip>${rat.skip}</skip>  <!-- DRILL-6320 License checks are intentionally skipped by default for development -->
```

Comment:
Is this necessary? Should not `<skip>` use `rat.skip` property by default?

Reply:
I originally tried `<skip>true</skip>` . However, this does not allow the skip to be overridden when the `-Drat.skip=false` argument is provided. It seems the only way to have an overridable default is to explicitly define a property with the default value.

Reply:
try with `<skip>${rat.skip}<skip>` removed. Keep `<rat.skip>true</rat.skip>` in properties.

Reply:
Your suggestion works. Made the change.